### ISSUE TITLE
Footer navigation's headings must be outside the `ul` 

### ISSUE THREAD

**Issue by joshkimux** [AIGAdc/dcdesignweek.org-11ty#15]
# Footer navigation's headings must be outside the `ul` [ ❗️ Must ]

## Defect level
Feedback will follow a must, should, consider feedback framework:

- ❗️ **Must for if the feedback must be applied.** Failure to follow this feedback may either cause harm to users, block a disabled person from completing an important task, or result in legal risk (fails to meet WCAG 2.1 AA compliance).
- ⚠️ **Should if the feedback is best practice.** Following this feedback will significantly improve the usability for disabled people or reduce the complexity of accessibility barriers.
- ✔️ **Consider for suggestions or enhancements.** Think of these suggestions as reducing the hundreds of papercuts and microaggressive barriers disabled people may face when navigating the web. Going above and beyond here will make us a leader in accessibility. 

## Description
Caught using axe-coconut. Issue was flagged as the `h4` is a direct child to the `ul` element in the footer.

<img width="1873" alt="Screen Shot 2022-05-01 at 2 22 27 PM" src="https://user-images.githubusercontent.com/14154792/166159201-74507e22-b9be-4570-bcf3-8ab205b9a716.png">

## Recommendation
Placing the `h4` outside and right before the `ul` will do the trick, but may require some css rejiggering

## Resources or References
[Deque University on list accessibility](https://dequeuniversity.com/rules/axe/4.2/list?application=coconut)
